sphinx-docs/Plugin-library.md

```diff
@@ -165,6 +165,22 @@ When this plugin has been loaded, CALDERA will start the HAProxy service on the
 
 CALDERA will **only** be available at https://[YOUR_IP]:8443 when using this plugin. All deployed agents should use the correct address to connect to CALDERA. 
 
+**Warning:** This plugin uses a default self-signed ssl certificate and key which is insecure. It is highly recommended that you generate your own before using the plugin to increase the safety of the system. See directions below.
+
+#### Using your own self-signed certificate
+In order to use this plugin securely, you need to generate your own self-signed certificate.
+The following directions explain how to generate and install your own certificate on a Linux or macOS system. 
+
+##### Directions:
+*Note: OpenSSL must be installed on your system*
+1. In the root CALDERA directory, navigate to `plugins/ssl`.
+2. In a terminal, paste the command `openssl req -x509 -newkey rsa:4096 -keyout conf/certificate.pem.key -out conf/certificate.pem -nodes` and press enter.
+3. This will prompt you for identifying details. Enter your country code when prompted. You may leave the rest blank by pressing enter.
```

Comment:
Remy latest feedback on https://github.com/mitre/ssl/pull/24, please add a step to append the private key to the certificate.pem file.